src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,79 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int OPERATION_TYPE_SYPPLY = 0;
+    private static final int OPERATION_TYPE_BUY = 1;
+    private static final int AMOUNT_OF_OPERATIONS = 2;
+    private static final int AMOUNT_FROM_STRING = 1;
+    private static final String COMA = ",";
+    private static final String OPERATION_SUPPLY = "supply";
+    private static final String OPERATION_BUY = "buy";
+    private static final String OPERATION_RESULT = "result";
 
     public void getStatistic(String fromFileName, String toFileName) {
+        int[] amounts = countTotalAmount(fromFileName);
+        String[] report = createReport(amounts[OPERATION_TYPE_SYPPLY], amounts[OPERATION_TYPE_BUY]);
+        writeToFile(toFileName, report);
+    }
+
+    private void writeToFile(String toFileName, String[] report) {
+        BufferedWriter bufferedWriter = null;
+        try {
```

Comment:
if you are using classes that implement an AutoCloseable interface, we should use it with try-with-resources.